### DISCUSSION TITLE
特定のキーを入力すると一部LEDが消灯する不具合

### DISCUSSION THREAD

**Discussion by 4km0k** [Yowkees/keyball#260] [Q&A] [answered]
keyball44を右トラックボール仕様で作成中です。
特定のキーを入力すると、一部LEDが消灯してしまう不具合が発生しています。

該当キー：左手側 3連親指キー 右（初期キーマップ「Lang 1」）
※cherry/ロープロソケットの両方で発生
※ソケットを外してショートさせた場合も同様
※スイッチを取り付けず、ピンセットでショートさせて入力

該当LED：6,28,29,30

・消えたLEDはUSBケーブルの抜き差しやリセットボタンを行うと復活します。
・LED消灯後もキー入力は問題ありません。
・入力すると必ず消灯するわけではなく、何度か入力すると消灯するLEDが増えます。発生頻度はLED6が一番高く、LED28が一番少ないです
・検証中、LED28が消灯せずに色だけ変わることが一度だけありました。
・左キーボード単体接続、右キーボード接続・TRRS経由の両方で発生します。

読みにくい文章で申し訳ないですが、お力添えいただけますと幸いです。

![IMG_20230626_223540](https://github.com/Yowkees/keyball/assets/137795693/7866341e-065e-4417-9172-ee18324659ee)
![IMG_20230626_223558](https://github.com/Yowkees/keyball/assets/137795693/215e71c2-769a-4956-8160-af8523b2f499)

**Comment by 4km0k** [comment]
【追記】
キースイッチを取り付けて再検証しましたが、スイッチ入力でLEDが消灯することはありませんでした。
該当ソケットを指で触ったり、ピンセットでショートさせると消灯するようです。
この挙動は正常でしょうか？

体の静電気を放電してから検証していますが、ノイズが乗ってしまいLEDが消灯しているようにも見えます。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

すみませんが私も経験のない現象です。
28,29,30が消えるのでしたら半田付けミスの可能性がありますが、６番だけとは。。
ただ、位置が近いLEDですので、何かの信号線にノイズが乗り、それを拾っている可能性はありますね。

基板を直接触るのはイレギュラーな挙動を引き起こしやすいですので、ひとまずキー操作で問題ないようでしたら、現状で様子を見ていただければ幸いです。

**Reply by 4km0k** [reply]
回答ありがとうございます。
一度組み上げてしばらく運用してみようと思います。
ご回答ありがとうございましたm(_ _)m
